modules/vorbis/audio_stream_ogg_vorbis.h

```diff
@@ -107,9 +107,9 @@ class AudioStreamOggVorbis : public AudioStream {
 	friend class AudioStreamPlaybackOggVorbis;
 
 	int channels = 1;
-	float length = 0.0;
+	double length = 0.0; // Changed to double because it's getter returns a double.
```

Comment:
Nit: here and below, you don't need to justify your changes with inline comments. The place for discussion of the changes themselves is the PR. Inline comments should be relevant for the next person to read the code, and they won't necessarily know about or care about what the broken code looked like before your changes.

Reply:
Removed the comments!